setup.cfg

```diff
@@ -15,6 +15,13 @@ per-file-ignores =
     *:F821
     */__init__.py: F401
 
+[mypy]
+strict_optional = True
+ignore_missing_imports = True
+files = flytekit/annotated/**/*.py,tests/flytekit/unit/annotated/**/*.py
```

Comment:
can we also add `flytekit/taskplugins/**` and `flytekit/types/**`

Reply:
Sure